### PR TITLE
OP-541 Standardize "accounting"

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-541.

### CODE REVIEW

File: src/main/java/org/isf/accounting/gui/PatientBillEdit.java [hunk 1]
```diff
@@ -988,90 +948,87 @@ private JButton getJButtonSave() {
...
+							thisBill.isPatient(),						//is a Patient?
+							thisBill.isPatient() ?
+								thisBill.getBillPatient() : null,			//Patient ID
+							thisBill.isPatient() ?
+								patientSelected.getName() :
+								jTextFieldPatient.getText(),			//Patient Name
+							paid ? "C" : "O",							//CLOSED or OPEN
+							total.doubleValue(),						//Total
+							balance.doubleValue(),						//Balance
+							user);										//User
```
Comment (mwithi) [inline src/main/java/org/isf/accounting/gui/PatientBillEdit.java:974]: these comments could be better aligned

File: src/main/java/org/isf/accounting/gui/PatientBillEdit.java [hunk 2]
```diff
@@ -988,90 +948,87 @@ private JButton getJButtonSave() {
...
+							thisBill.isPatient(),						//is a Patient?
+							thisBill.isPatient() ?
+								thisBill.getBillPatient() : null,			//Patient ID
+							thisBill.isPatient() ?
+								thisBill.getPatName() :
+								jTextFieldPatient.getText(),			//Patient Name
+							paid ? "C" : "O",							//CLOSED or OPEN
+							total.doubleValue(),						//Total
+							balance.doubleValue(),						//Balance
+							user);										//User
```
Comment (mwithi) [inline src/main/java/org/isf/accounting/gui/PatientBillEdit.java:1005]: these comments could be better aligned

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

File: src/main/java/org/isf/accounting/gui/PatientBillEdit.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/accounting/gui/PatientBillEdit.java:974]: done

File: src/main/java/org/isf/accounting/gui/PatientBillEdit.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/accounting/gui/PatientBillEdit.java]: done

**Pull request merged**